tests/list_test.cpp

```diff
@@ -141,6 +140,7 @@ TEST_CASE( "list basics", "[list]" )
         SECTION( "reverse iterator count/access" ) {
```

Comment:
Either this is lying or the loop below is wrong.

Reply:
Yeah; this is a copy/paste error.  Will fix.